cpp/node_module/node_module.cpp

```diff
@@ -6,6 +6,12 @@ extern "C" int mgp_init_module(struct mgp_module *module, struct mgp_memory *mem
   try {
     mgp::memory = memory;
 
+    AddProcedure(
+        Node::RelationshipTypes, Node::kProcedureRelationshipTypes, mgp::ProcedureType::Read,
+        {mgp::Parameter(Node::kRelationshipTypesArg1, mgp::Type::Node),
+         mgp::Parameter(Node::kRelationshipTypesArg2, {mgp::Type::List, mgp::Type::String}, mgp::Value(mgp::List{}))},
+        {mgp::Return(Node::kResultRelationshipTypes, {mgp::Type::List, mgp::Type::String})}, module, memory);
```

Comment:
Going off APOC documentation ([link 1](https://neo4j.com/labs/apoc/4.1/overview/apoc.node/apoc.node.relationship.types/) & [link 2](https://neo4j-contrib.github.io/neo4j-apoc-procedures/3.5/nodes-relationships/node-functions/)), I see that the second argument is not a list of strings, but rather a string with relationship types joined by `"|"`, e.g. `"FRIEND|MENTORS>|<REPORTS_TO"`. Can you confirm if the difference in implementation is OK from a  compatibility perspective?

Reply:
@antoniofilipovic I'd also appreciate your opinion

Reply:
We decided to use a list instead of splitting by | for simplicity.

Reply:
Yes, we decided to use list for simplicity